substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/logging/JfrLogConfiguration.java

```diff
@@ -95,7 +97,7 @@ private static void verifySelections(JfrLogSelection[] selections) {
             }
         }
     }
-
+    @SuppressWarnings("checkstyle:LocalVariableName")
```

Comment:
We don't want to suppress checkstyle warnings unless there is a really good reason for that. In this particular case, it should be sufficient to just rename the locals.